docs/statespace/filters.rst

```diff
@@ -2,14 +2,11 @@
 Kalman Filters
 ********************
 
-.. automodule:: pymc_experimental.statespace.filters
+.. automodule:: pymc_extras.statespace.filters
 .. autosummary::
    :toctree: generated
 
     StandardFilter
     UnivariateFilter
-    SteadyStateFilter
     KalmanSmoother
-    SingleTimeseriesFilter
-    CholeskyFilter
```

Comment:
Unrelated changes? Did you merge main instead of rebasing? Makes it a bit hard to check

Reply:
I removed these because they did not seem to exist in the code base, so tests were failing.

Reply:
The cholesky, was renamed to SquareRootFilter. The others were removed as you said